src/node/internal/internal_http_outgoing.ts

```diff
@@ -3,84 +3,502 @@
 //     https://opensource.org/licenses/Apache-2.0
 // Copyright Joyent and Node contributors. All rights reserved. MIT license.
 
+/* eslint-disable @typescript-eslint/no-deprecated */
```

Comment:
Which addition would trigger the deprecated warning?

Reply:
`this.finished` and other attributes which is deprecated in types/node package.

Reply:
Might benefit from a comment about it. Not critical tho.

Reply:
You're right. A comment will be nice. I'll add.